### ISSUE TITLE
Broken Frame Order

### ISSUE THREAD

**Issue by UltraFaceguy** [miniaturepets/issues#177]
* Server Software: git-Paper-146 (PaperSpigot)
* Plugin Version:  Version 2.3.9
* Dump File: https://pastebin.com/UgT52xYk

I noticed that the animation is skipping frame 3 and running it at the end for some reason
F1 - headPos: "{0 , 0 , 0}"
F2 - headPos: "{0 , 90 , 0}"
F3 - headPos: "{0 , 0 , 0}"
F4 - headPos: "{0 , 180 , 0}"
F5 - headPos: "{0 , 270 , 0}"

this performs a 360 degree rotation smoothly. because F3 executes ATFER F5.
order: F1 -> F2 -> F4 -> F5 -> F3

mpet in question: https://file.io/n5mklRq1

**Comment by danielnaaman** [comment]
Does this still happen with the latest PaperMC and Miniature Pets versions?

**Comment by UltraFaceguy** [comment]
Yes. Happens in the latest versions. https://www.dropbox.com/s/3jwx9a13lcocm6u/SillyGlobe.mpet?dl=0

There's the pet file again, old link seems broken. See below for animations for idle and moving animation. You can see the weirdness in the moving one, the frame order is still as posted above:
F1 - headPos: "{0 , 0 , 0}"
F2 - headPos: "{0 , 90 , 0}"
F3 - headPos: "{0 , 0 , 0}"
F4 - headPos: "{0 , 180 , 0}"
F5 - headPos: "{0 , 270 , 0}"

![movinganim](https://user-images.githubusercontent.com/1491435/92309512-520c3d00-ef74-11ea-95a9-fc887fffaabb.gif)
![idleanim](https://user-images.githubusercontent.com/1491435/92309447-9d721b80-ef73-11ea-9268-3a89a0ecad4c.gif)

**Comment by danielnaaman** [comment]
Fixed, will be seen in the next version of Miniature Pets.
